src/ui/osx/TogglDesktop/MainWindowController.m

```diff
@@ -80,11 +80,6 @@ - (void)windowDidLoad
 {
 	[super windowDidLoad];
 
-	// Clean window titlebar
-	self.window.titleVisibility = NSWindowTitleHidden;
-	self.window.titlebarAppearsTransparent = YES;
-	self.window.styleMask |= NSFullSizeContentViewWindowMask;
```

Comment:
The problem is here. As we set the `NSFullSizeContentViewWindowMask` for a main Window after initialization time (-windowDidLoad), so the content size will accidentally get smaller.

Thus, Override it in XIB is better way.